GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -175,26 +198,118 @@ public void executeImport(File file, String serverBase, Map<String, String> crit
                 threadPool.shutdown();
             cleanup();
         }
+        
+        //TODO: we could send result to server for the notification tab
```

Comment:
Can you make a note of this under the https://github.com/akvo/akvo-product-design/issues/218 repo and remove the comment?

Reply:
Removing it, this already what happens from the Clojure code.